app.rb

```diff
@@ -93,6 +93,7 @@ class Tint < Sinatra::Base
       else
         stream_content(file_path, &tmp.method(:puts))
       end
+      tmp.flush
```

Comment:
Forces any in-memory write buffers to be written to disk. Without this on my laptop saves always result in an empty file.